### PR TITLE
firmware: intel-microcode: update to 20210608

### PR DESCRIPTION
**Pull request by nasbdh9**
intel-microcode: update to 20210608

Run test: x86_64

Signed-off-by: Tan Zien nabsdh9@gmail.com

### CODE REVIEW

File: package/firmware/intel-microcode/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=intel-microcode
-PKG_VERSION:=20200616
-PKG_RELEASE:=1
+PKG_VERSION:=20210608
+PKG_RELEASE:=2
```
Comment (jospezial) [inline package/firmware/intel-microcode/Makefile:12]: I think PKG_RELEASE should increase only if something is changed on the same version.

Comment (nasbdh9) [reply package/firmware/intel-microcode/Makefile:12]: Just to match the debian upstream

Comment (Borromini) [inline package/firmware/intel-microcode/Makefile:12]: What @jospezial meant was you should not be changing PKG_RELEASE from 1 to 2 snice you are changing the PKG_VERSION variable already. So PKG_RELEASE stays 1.

Comment (nasbdh9) [reply package/firmware/intel-microcode/Makefile:12]: Just to match the debian upstream

Comment (hauke) [reply package/firmware/intel-microcode/Makefile:12]: The PKG_RELEASE variable is OpenWrt specific and unrelated to upstream. We update it when some changes are done to a package without change the version number like adding an extra patch.

Comment (nasbdh9) [reply package/firmware/intel-microcode/Makefile]: Please check the new commit, I found that debian will only keep the latest version of the non-release version of the file, switch to the file with the release suffix, cancel the use of variables to get the file, and avoid the failure of upstream links

**Comment by chunkeey** [comment]
Merged. Thanks!

**Pull request closed without merging**